styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -309,6 +302,20 @@ public void eventReceived(Action action, Pod pod) {
       }
     }
 
+    private void logEvent(Action action, Pod pod) {
+      final String podName = pod.getMetadata().getName();
+      LOG.info("Pod event for {} at resource version {}", podName, lastResourceVersion);
+      LOG.info("Action: {}", action);
+      final String workflowInstance = pod.getMetadata().getAnnotations()
+              .getOrDefault(KubernetesDockerRunner.STYX_WORKFLOW_INSTANCE_ANNOTATION, "N/A");
+      LOG.info("Workflow instance: {}", workflowInstance);
```

Comment:
Thanks! The "Status" is already quite messy now, so I thought it would be better having them separated.